app/src/main/java/com/android/sample/ui/camera/CameraPreview.kt

```diff
@@ -72,33 +77,51 @@ fun Carousel(openDialog: () -> Unit, itemsList: List<Bitmap>, deleteImage: (Bitm
   Row(modifier = Modifier.padding(STANDARD_PADDING.dp).height(120.dp)) {
     FloatingActionButton(
         content = {
-          Icon(imageVector = Icons.Outlined.AddCircle, contentDescription = "Add a new image")
+          Icon(
+              imageVector = Icons.Filled.AddAPhoto, // Using a more specific icon for adding photos
+              contentDescription = "Add a new image", // Providing an accessible description
+              modifier = Modifier.size(MEDIUM_FONTSIZE.dp) // Icon size can be adjusted as needed
+              )
         },
         onClick = openDialog,
         modifier =
             Modifier.size(LARGE_BUTTON_HEIGHT.dp)
                 .background(Color(WHITE_COLOR))
                 .testTag("addImageButton"),
-    )
+        contentColor = Color(MAIN_COLOR_DARK),
+        containerColor = Color(MAIN_BACKGROUND_BUTTON))
     LazyRow {
       items(itemsList.size) { bitmap ->
-        Card(modifier = Modifier.padding(SMALL_PADDING.dp)) {
-          Image(
-              bitmap = itemsList[bitmap].resize(IMAGE_SIZE).asImageBitmap(),
-              contentDescription = "Selected Image",
-              modifier = Modifier.size(IMAGE_SIZE.dp))
+        Box() {
+          Card(
+              modifier = Modifier.padding(SMALL_PADDING.dp),
+              colors =
+                  CardDefaults.cardColors(
+                      containerColor = Color(MAIN_BACKGROUND_BUTTON),
+                      contentColor = Color(MAIN_COLOR_DARK)),
+              border = BorderStroke(3.dp, Color(MAIN_BACKGROUND_BUTTON))) {
+                Image(
+                    bitmap = itemsList[bitmap].resize(IMAGE_SIZE).asImageBitmap(),
+                    contentDescription = "Selected Image",
+                    modifier = Modifier.size(IMAGE_SIZE.dp),
+                    contentScale = ContentScale.FillBounds)
+              }
           IconButton(
               onClick = { deleteImage(itemsList[bitmap]) },
               modifier =
-                  Modifier.width(MEDIUM_IMAGE_SIZE.dp)
-                      .height(MEDIUM_IMAGE_SIZE.dp)
-                      .align(Alignment.End)
-                      .testTag("removeImageButton")) {
-                Icon(
-                    imageVector = Icons.Filled.Close,
-                    contentDescription = "Remove Image",
-                    tint = Color.Black)
-              }
+                  Modifier.size(48.dp) // Appropriate size for touch target
+                      .align(Alignment.TopEnd) // Align to the top end of the Box, not the Card
+                      .background(
+                          color = Color.White.copy(alpha = 0.0F), // White background for contrast
+                          shape = CircleShape // Circular background for the delete button
+                          ), // Optional: Add a background for better visibility
```

Comment:
Next time, if you remove a testTag, pay attention to put it back ! For now it is fine as it is not used elsewhere.